mmv1/third_party/terraform/website/docs/r/apigee_api_deployment.html.markdown

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "Apigee"
+description: |-
+  Deploys a revision of an API proxy.
+---
+
+# google_apigee_api_deployment
+
+Deploys a revision of an API proxy.
+
+To get more information about API proxy deployments, see:
+
+* [API documentation](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.apis.revisions.deployments)
+* How-to Guides
+    * [apis.revisions.deployments](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.apis.revisions.deployments)
+
+## Example usage
+
+```hcl
+data "archive_file" "bundle" {
+  type             = "zip"
+  source_dir       = "${path.module}/bundle"
+  output_path      = "${path.module}/bundle.zip"
+  output_file_mode = "0644"
+}
+
+resource "google_apigee_api" "api" {
+  name          = "api1"
+  org_id        = var.org_id
+  config_bundle = data.archive_file.bundle.output_path
+}
+
+resource "google_apigee_api_deployment" "api_deployment" {
+  environment = var.env_id
+  org_id      = google_apigee_api.api.org_id
+  revision    = google_apigee_api.api.revision[length(google_apigee_api.api.revision)-1]
+  api_id      = google_apigee_api.api.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org_id` -
+  (Required)
+  The Apigee Organization associated with the API proxy
+
+* `environment` -
+  (Required)
+  The resource ID of the environment.
+
+* `api_id` -
+  (Required)
+  Id of the API proxy to be deployed.
+
+* `revision` -
+  (Required)
+  Revision of the API proxy to be deployed.
+
+
+- - -
+
+
+* `service_account` -
+  (Optional)
+  The service account represents the identity of the deployed proxy, and determines what permissions it has. The format must be {ACCOUNT_ID}@{PROJECT}.iam.gserviceaccount.com.
+
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `id` - an identifier for the resource with format `organizations/{{org_id}}/environments/{{environment}}/apis/{{api_id}}/revisions/{{revision}}/deployments`
+
+
+## Timeouts
+
+This resource provides the following
+[Timeouts](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts) configuration options:
+
+- `create` - Default is 20 minutes.
+- `update` - Default is 20 minutes.
+- `delete` - Default is 20 minutes.
+
+## Import
+
+
+An API proxy deployment can be imported using any of these accepted formats:
+
+* `organizations/{{org_id}}/environments/{{environment}}/apis/{{api_id}}/revisions/{{revision}}/deployments/{{name}}`
+* `{{org_id}}/{{environment}}/{{sharedflow_id}}/{{revision}}/{{name}}`
```

Comment:
s/sharedflow_id/api_id/ ?

Similar for various shared flow references below this on the page.

Reply:
updated the references